dist/Time-HiRes/HiRes.xs

```diff
@@ -20,6 +20,7 @@ extern "C" {
 #include "XSUB.h"
 #include "reentr.h"
 #ifdef USE_PPPORT_H
+#define NEED_ck_warner
 #  include "ppport.h"
```

Comment:
note that `ppport.h` is not included in the distro

Reply:
https://github.com/Perl/perl5/pull/19640/files